### PR TITLE
WIP: Hmis096a tb register summary

### PR DESCRIPTION
**Pull request by moshonk**
Please find this @ssmusoke 
Updated data exchange xml file with missing concepts
Fixed markup issue that was breaking javascript code

### CODE REVIEW

**Comment by ssmusoke** [comment]
@moshonk Do see email on concepts can you fix it for all the concepts in this PR so that I can review

**Comment by moshonk** [comment]
@ssmusoke I have removed the fields as requested @ningosi

File: omod/src/main/webapp/resources/htmlforms/096a-HealthUnitTBRegister-SummaryPage.xml [hunk 1]
```diff
@@ -862,7 +865,7 @@
               	<label>Treatment Outcome</label>
               	<obs 	conceptId="99423" 
               			answerConceptIds="159791AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,90306,159874AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1366,5240,160737"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/096a-HealthUnitTBRegister-SummaryPage.xml]: @moshonk These should be concept UUIDs

File: api/src/main/resources/metadata/Concepts_MCH-1.xml [hunk 2]
```diff
@@ -114,54 +114,54 @@
...
+  <concept_description concept_description_id="18317" concept_id="164948" description="Age of baby in weeks" locale="en" creator="1" date_created="2017-05-23 16:12:38.0" uuid="434b9b32-8963-47d4-83e8-54f06b520c27"/>
+  <concept_description concept_description_id="18318" concept_id="164954" description="Referring clininc" locale="en" creator="1" date_created="2017-05-23 17:30:14.0" uuid="4981eb68-bd67-471b-b91e-bed2b1e02935"/>
+  <concept_description concept_description_id="2951" concept_id="99267" description="Place patient has been referred to for farther care provision." locale="en" creator="1" date_created="2011-03-16 06:20:47.0" uuid="6128c21d-24c8-4700-8f0e-49cd834c7ec9"/>
+  <concept_description concept_description_id="3030" concept_id="99360" description="Referal of babies on PNC Visit" locale="en" creator="1" date_created="2012-06-27 12:31:13.0" uuid="95e60222-7718-4d04-a4e4-e9dc044de759"/>
+  <concept_description concept_description_id="3031" concept_id="99361" description="Referal of babies on PNC Visit" locale="en" creator="1" date_created="2012-06-27 12:32:07.0" date_changed="2012-06-27 12:34:32.0" uuid="1cb408b7-6176-410f-a43a-31cf2471d135"/>
```
Comment (ssmusoke) [inline api/src/main/resources/metadata/Concepts_MCH-1.xml]: @moshonk Remove all date_changed fields

File: api/src/main/resources/metadata/Concepts_MCH-1.xml [hunk 3]
```diff
@@ -114,54 +114,54 @@
...
   <concept_name concept_id="17" name="DIPTHERIA TETANUS BOOSTER" locale="en" creator="1" date_created="2004-01-01 00:00:00.0" concept_name_id="19" voided="false" voided_by="1" uuid="19BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
   <concept_name concept_id="17" name="TD BOOSTER" locale="en" creator="1" date_created="2005-02-12 11:57:51.0" concept_name_id="98729" voided="false" voided_by="1" void_reason="" uuid="98729BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" locale_preferred="false"/>
   <concept_name concept_id="17" name="INJECTION DE RAPPEL: DIPTH&#201;RIE T&#201;TANOS" locale="fr" creator="1" date_created="2010-08-02 19:38:31.0" concept_name_id="106374" voided="false" voided_by="1" uuid="106374BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" concept_name_type="FULLY_SPECIFIED" locale_preferred="true"/>
```
Comment (ssmusoke) [inline api/src/main/resources/metadata/Concepts_MCH-1.xml]: @moshonk Remove all voided_by="1" fields

**Comment by moshonk** [comment]
@ssmusoke I have worked on the following, kindly review:
Updated relevant  Post submission actions for the TB and MCH forms
Removed duplicated concept(id 1646) and its respective references
Cleaned up the concepts by removing the changed_by, date_changed and retired_by fields 
Changed conceptId attribute of obs tags to reference Concept UUID instead of conceptId
cc @ningosi

**Comment by ssmusoke** [comment]
@moshonk When I enter only the following data: TB unit numbers, Patient type, No for whether the patient is a health worker, then put an outcome Cured and an outcome date, the form does not save. It shows that there are validation errors but does not show what validations are missing.

**Pull request merged**